components/ParsePOSPayment/Receive-Invoice.tsx

```diff
@@ -248,6 +298,59 @@ function ReceiveInvoice({ recipientWalletCurrency, walletId, state, dispatch }:
         paymentAmount={paymentAmount}
         dispatch={dispatch}
       />
+      {openModal && (
+        <Modal
+          show={openModal}
+          onHide={() => setOpenModal(false)}
+          aria-labelledby="contained-modal-title-vcenter"
+          centered
+        >
+          <Modal.Header className={styles.modal_header} closeButton>
+            You can payment with
+          </Modal.Header>
+          <Modal.Body>
+            <div className={styles.modal_item_wrapper}>
+              <Image
+                src="/manifest-logo/logo144x144.svg"
```

Comment:
Please use the bbw app logo here. this is the cash register's logo
```suggestion
                src="/manifest-logo/logo144x144.svg"
```